packages/pds-ember/addon/components/pds/external-link/index.js

```diff
@@ -8,7 +8,26 @@ import Component from '@glimmer/component'
  * Controls visibility of the trailing icon.
  *
  * @argument hideIcon
- * @type {boolean}
+ * @type {?boolean}
```

Comment:
just curious - is this a new way to declare types? (in my mind I am thinking this relates to  `this.connectionTargets?.peeringTargets.length` of which i cannot remember the technical term used to describe this right now)

Reply:
I had placed it there to denote that it was optional but I think there's actually might be an `@optional` that can be used

Reply:
ah okay I was just wondering.  Also now I am remembering, I want to say  I thought we could just say 'optional' or not include 'required' (going off memory - I am going to look just for my own purposes).  Thank you!